tests/src/lib.rs

```diff
@@ -212,7 +212,7 @@ fn map_wasm_value(
 }
 
 impl<'a, 'b, 'hooks> WasmtimeHelper<'a, 'b, 'hooks> {
-    pub fn new(
+    pub fn new_from_file(
```

Comment:
Could you please explain why the renaming? Maybe adding a doc comments?

Reply:
Because we will likely have some very short clarity snippets to test and then we can introduce a `new_from_str` and inline the clarity.

Reply:
Added that now actually, as well as doc comments.